src/components/MatPassContent.tsx

```
@@ -906,38 +1136,51 @@ export default function MatPassContent() {
+        !pkg.name.toLowerCase().includes("co-work") &&
+        pkg.name.toLowerCase().includes("acro")
     )
     .sort((a: BundleType, b: BundleType) => a.price - b.price)
     .map((pkg: BundleType) => {
       const packageNumber = getPackageNumber(pkg.name);
       const isDouble = pkg.price === 160 || pkg.price === 530;
       const validId = parseInt(pkg.id);
       return {
-        id: isNaN(validId) ? (isDouble ? 9000 + packageNumber : 8000 + packageNumber) : validId,
+        id: isNaN(validId)
+          ? isDouble
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)